src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  visibleModal: Todo;
+  setVisibleModal: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  visibleModal, setVisibleModal,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
```

Comment:
You are not handling errors in your asynchronous function. This can lead to uncaught exceptions if the promise rejects. Always handle errors in promises by using a .catch() block or a try/catch statement if inside an async function.